app/graphql/types/query_types.rb

```diff
@@ -34,5 +34,26 @@ def wells(plate_id: nil)
     def plates
       Plate.all
     end
+
+    field :libraries, [Types::Outputs::LibraryType], null: false do
+      description 'Find all Libraries'
+    end
+
+    def libraries
+      # ONT::Library.all    
+      # name = 'plate_barcode-poolnumber'
+      [
+        { id: 1, tube_barcode: 'TRAC-2-1', plate_barcode: 'TRAC-1-1', pool: 1,
+          name: 'TRAC-1-1-1', wells: 'A1-H3', tag_set: 24 },
+        { id: 2, tube_barcode: 'TRAC-2-2', plate_barcode: 'TRAC-1-1', pool: 2,
+          name: 'TRAC-1-1-2', wells: 'A4-H6', tag_set: 24 },
+        { id: 3, tube_barcode: 'TRAC-2-3', plate_barcode: 'TRAC-1-1', pool: 3,
+          name: 'TRAC-1-1-3', wells: 'A7-H9', tag_set: 24 },
+        { id: 4, tube_barcode: 'TRAC-2-4', plate_barcode: 'TRAC-1-1', pool: 4,
+          name: 'TRAC-1-1-4', wells: 'A10-H12', tag_set: 24 },
+        { id: 5, tube_barcode: 'TRAC-2-5', plate_barcode: 'TRAC-1-2', pool: 1,
+          name: 'TRAC-1-2-1', wells: 'A1-H12', tag_set: 96 }
+      ]
```

Comment:
I don't think GraphQL will cope well with this array being returned.  It uses the type of the value here to know which output type to use to create the JSON response.  We don't have an output type for arrays.  It might just handle this without an output type specified, but I can't be sure that's the case.